ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-path-selection.j2

```diff
@@ -0,0 +1,120 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - router path-selection #}
+{% if router_path_selection is arista.avd.defined %}
+
+### Router Path-selection
+
+#### Router Path-selection Summary
+
+| Setting | Value |
+| ------  | ----- |
+{%     if router_path_selection.peer_dynamic_source is arista.avd.defined %}
```

Comment:
```suggestion
{%     if router_path_selection.peer_dynamic_source is arista.avd.defined %}

#### Router Path-selection Summary

| Setting | Value |
| ------  | ----- |
```

Reply:
will do it for now as there is only one setting but this won't work in the future if there are more

Reply:
[5904680](https://github.com/aristanetworks/ansible-avd/pull/3203/commits/59046801f25e072cff6a5a5fa1c76f19c2429c7c)